### PR TITLE
Don't show teacher-facing markdown or video hints in non-english locales

### PR DESCRIPTION
**Pull request by joshlory**
These were an experiment for a single stage. We don't plan to use these more places going forward, or invest in translating these videos.

See also: PR https://github.com/code-dot-org/code-dot-org/pull/19009.

### CODE REVIEW

**Comment by joshlory** [comment]
[Don’t show video hints in non-english locales](https://trello.com/c/BaqCeyAJ/236-dont-show-video-hints-in-non-english-locales)

**Comment by joshlory** [comment]
[Hide teacher-facing markdown for non-english locales](https://trello.com/c/HJzeRvHx/239-hide-teacher-facing-markdown-for-non-english-locales)

File: dashboard/app/models/levels/blockly.rb [hunk 1]
```diff
@@ -371,7 +371,7 @@ def localized_authored_hints
       return unless authored_hints_key
 
       localized_hints = JSON.parse(authored_hints).map do |hint|
-        next if hint['hint_markdown'].nil? || hint['hint_id'].nil?
+        next if hint['hint_markdown'].nil? || hint['hint_id'].nil? || hint['hint_video'].present?
```
Comment (Hamms) [inline dashboard/app/models/levels/blockly.rb]: I'd love to see a comment here explaining this change

File: dashboard/app/helpers/levels_helper.rb [hunk 2]
```diff
@@ -220,7 +220,7 @@ def app_options
       end
 
     # Blockly caches level properties, whereas this field depends on the user
-    @app_options['teacherMarkdown'] = @level.properties['teacher_markdown'] if current_user.try(:authorized_teacher?)
+    @app_options['teacherMarkdown'] = @level.properties['teacher_markdown'] if current_user.try(:authorized_teacher?) && I18n.en?
```
Comment (Hamms) [inline dashboard/app/helpers/levels_helper.rb:223]: huh. I didn't know this method was a thing! I've been doing `I18n.locale == I18n.default_locale` for this.

**Pull request merged**